### PR TITLE
Chrish/click handler

### PR DESCRIPTION
**Pull request by clhenrick**
We need a way of intercepting click events on charts, so that the nearest data point can be intercepted and passed to the chart's parent for things like crossfiltering in Immerse. This PR starts to address this.

### CODE REVIEW

File: src/charts/tooltip.js [hunk 1]
```diff
@@ -161,9 +161,8 @@ export default function Tooltip (_container, _isLegend = false) {
           }
         ]
 
-        if (_isLegend) {
-          legendData.push({key: "tooltip-label", value: d[keys.LABEL]})
-        }
+        // TO DO: enable this via a config option?
```
Comment (biovisualize) [inline src/charts/tooltip.js]: I think this doesn't belong to this PR

**Comment by biovisualize** [comment]
That looks like the right way. I didn't check if it shares enough code with other event handlers to be made reusable.

File: src/charts/tooltip.js (hunk 1 above)
Comment (clhenrick) [reply src/charts/tooltip.js]: correct, it was left over when I branched off #31, will remove it
